ports/servoshell/desktop/app.rs

```diff
@@ -406,7 +410,19 @@ impl ApplicationHandler<WakerEvent> for App {
         self.handle_events_with_winit(event_loop, window);
     }
 
-    fn user_event(&mut self, event_loop: &ActiveEventLoop, event: WakerEvent) {
+    fn user_event(&mut self, event_loop: &ActiveEventLoop, event: AppEvent) {
+        if let AppEvent::Accessibility(ref e) = event {
+            if let Some(ref mut minibrowser) = self.minibrowser {
+                if minibrowser.handle_accesskit_event(&e.window_event) {
+                    let Some(window) = self.windows.get(&e.window_id) else {
+                        return;
+                    };
+                    window.winit_window().unwrap().request_redraw();
+                    return;
+                }
+            }
+        }
```

Comment:
Maybe it's possible to reduce the indentation here:

```suggestion
        if let AppEvent::Accessibility(ref event) = event {
           if let (window) = self.minibrowser
               .as_mut()
               .and_then(|minibrowser| {
                   minibrowser.handle_accesskit_event(&e.window_event));
                   self.windows.get(&event.window_id);
                }) {
                    window.winit_window().unwrap().request_redraw();
               }
               return;
```